### DISCUSSION TITLE
Windows installation source

### DISCUSSION THREAD

**Discussion by snibgo** [ImageMagick/ImageMagick#2986] [General]
The documentation https://imagemagick.org/script/install-source.php used to direct us to download a zip file, but now says we should:
```
git clone git@github.com:ImageMagick/ImageMagick-Windows.git ImageMagick-Windows-7
```

But that doesn't work. I get:
```
Cloning into 'ImageMagick-Windows-7'...
The authenticity of host 'github.com (140.82.121.3)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no/[fingerprint])? y
Please type 'yes', 'no' or the fingerprint: yes
Warning: Permanently added 'github.com,140.82.121.3' (RSA) to the list of known
hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

**Comment by urban-warrior** [comment]
We updated the instructions to include an alternative download.  Recommended practice is `git`, however.

**Comment by snibgo** [comment]
"git clone" now works, without a "permission denied" error.

Thanks.

**Comment by snibgo** [comment]
The section "Install from Unix Source" says we should run:
```
$ git clone https://github.com/ImageMagick/ImageMagick.git ImageMagick-7.0.10
```
This works fine, downloading IM (but with no delegates).


The section "Install from Windows Source" says we should run:
```
git clone https://github.com/ImageMagick/ImageMagick-Windows.git ImageMagick-Windows-7
```
This downloads a small number of files. Then we should run:
```
CloneRepositories.cmd
```
This fails for me because it can't find bash.exe. For some reason it doesn't look on my system path. The command "where bash" would find it, at "C:\cygwin64\bin\bash.exe".

So I can simply type the command:
```
bash -i -c "./CloneRepositories.sh"
```

But that gives errors:
```
Cloning
fatal: repository 'aom/.git' does not exist
fatal: not a git repository (or any of the parent directories): .git

Cloning
fatal: repository 'brotli/.git' does not exist
fatal: not a git repository (or any of the parent directories): .git

Cloning
fatal: repository 'bzlib/.git' does not exist
fatal: not a git repository (or any of the parent directories): .git
```
... and so on. I don't know how to fix that.

The ZIP file looks good. 110MB, so I suppose it includes the common delegates.
